### PR TITLE
remove datalodaer

### PR DESCRIPTION
**Pull request by mickhansen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,18 @@ resolver(SequelizeModel, {
     result.sort(/* Custom sort function */);
     return result;
   },
+
+  /*
+   * Transfer fields from the graphql contet to the options passed to model calls
```
Comment (janmeier) [inline README.md]: contet

File: src/resolver.js [hunk 2]
```diff
@@ -58,6 +54,10 @@ function resolverFactory(target, options = {}) {
     findOptions.logging = findOptions.logging || context.logging;
     findOptions.graphqlContext = context;
 
+    _.each(contextToOptions, (as, key) => {
```
Comment (janmeier) [inline src/resolver.js]: _.assign

**Comment by janmeier** [comment]
Since this is a build repo you might want to write a couple of lines about this change :P

**Comment by janmeier** [comment]
Paging

https://github.com/mickhansen/graphql-sequelize/issues/534
https://github.com/mickhansen/graphql-sequelize/issues/509
https://github.com/mickhansen/graphql-sequelize/issues/455
https://github.com/mickhansen/graphql-sequelize/issues/418

File: src/resolver.js (hunk 2 above)
Comment (mickhansen) [reply src/resolver.js]: Can't use assign as aliasing is supported

**Pull request merged**